azure-iot-device/azure/iot/device/common/pipeline/operation_flow.py

```diff
@@ -241,3 +241,22 @@ def complete_op(stage, op):
             exc_info=e,
         )
         stage.pipeline_root.unhandled_error_handler(e)
+
+
+def push_pipeline_event_up(stage, event):
```

Comment:
>push_pipeline_event_up [](start = 4, length = 22)

I think there should be symmetry with this and  `pass_op_to_next_stage`, given that they're similar functions, they should have similar naming conventions.

They currently have:

1) Different verbs: `push` vs `pass`
2) Different styled reference to the pipeline object: `pipeline_event` vs `op`
3) Different styled reference to destination: `up` vs `next_stage`